src/PlaywrightSharp/IFrame.cs

```diff
@@ -433,7 +433,7 @@ public interface IFrame
         /// <param name="noWaitAfter">Actions that initiate navigations are waiting for these navigations to happen and for pages to start loading.
         /// You can opt out of waiting via setting this flag. You would only need this option in the exceptional cases such as navigating to inaccessible pages. Defaults to false.</param>
         /// <returns>A <see cref="Task"/> that completes when the element matching <paramref name="selector"/> is successfully double clicked.</returns>
-        Task DblClickAsync(
+        Task DblclickAsync(
```

Comment:
This is a generator problem. It should be `DblClickAsync`, so we should definitely have an upstream PR that adds a `alias-csharp: DblClickAsync` into the file.

Reply:
https://github.com/microsoft/playwright/pull/5899